lib/rails-settings/settings.rb

```diff
@@ -60,6 +60,8 @@ def get_all(starting_with = nil)
         vars.each do |record|
           result[record.var] = record.value
         end
+        vars.merge! @@defaults.slice(*(@@defaults.keys - result.keys))
```

Comment:
Shouldn't this be `results.merge!` so that these values are returned?

Also, thanks for cc'ing me, I really appreciate it. :)